### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Henryhaulka**
In this Project (milestone 1 ), we:

- Cloned the Newsweek Homepage.
- Used Bootstrap
- Created the Footer and header.

### CODE REVIEW

**Review by MakZ — COMMENTED** [review]
Fixed Linters error

File: index.html [hunk 1]
```diff
@@ -72,5 +75,49 @@
...
+          <a href="">Contact Us</a> | <a href="">Editorial Guidelines</a> |
+          <a href="">Advertise</a> | <a href="">Copyright</a> |<a href="">
+            Terms & Conditions</a
+          >
+          | <a href="">Privacy Policy</a> | <a href=""> Cookies Policies</a> |
+          <a href=""> Terms of Sales</a> |<a href=""> Archives</a> |
+          <a href=""> Announcement </a> | <a href="">Consent Preferences</a>
+        </div>
+      </div>
+    </footer>
```
Comment (BrittanyBlake) [inline index.html]: - [x] Your footer should stick to the bottom of the page even if there is no content. You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`

File: index.html [hunk 2]
```diff
@@ -21,17 +21,19 @@
   <body>
     <header class="container">
       <div class="d-flex head1 justify-content-between text-white">
-        <div class="p-2 d-flex align-items-end ml-3">
+        <div id="date" class="p-2 d-flex align-items-end ml-3">
           <p>Wed, jan 20,2021</p>
         </div>
         <div class="p-2 d-flex justify-content-between">
           <img
+            class="logo-header-img"
```
Comment (BrittanyBlake) [inline index.html]: - [x] The logo image should have a link tag

File: index.html [hunk 3]
```diff
@@ -21,17 +21,19 @@
   <body>
     <header class="container">
```
Comment (BrittanyBlake) [inline index.html]: - [x] The navigation should use the `<nav>` tag
 screenshot from requirements 👇🏾 
Screen Shot 2021-03-01 at 7 23 59 PM

File: index.html [hunk 4]
```diff
@@ -63,7 +66,7 @@
           <i class="fas fa-search"></i>
```
Comment (BrittanyBlake) [inline index.html]: - [x] Your search form field should use the form tag 👇🏾 
Screen Shot 2021-03-01 at 7 28 13 PM

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi Team! 👋 

You have done an amazing job with this milestone! It looks really good 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Some of your css styles can be done with bootstrap classes. Take a look at your css file again and use this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) to see what can be done with bootstrap instead. 
- [x] Your footer should stick to the bottom of the page even if there is no content.


****

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @Henryhaulka and @MakZ  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, this milestone is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] You still have a lot of CSS codes that you can implement using bootstrap classes: for example, don't do flex manually, you can use bootstrap to achieve the flex functionalities, properties like `display: none;` can be achieved by bootstrap utility classes. You even have more CSS codes such as margins, paddings, widths. heights which can be implemented using bootstrap utility classes. Please pause a bit and take your time again. Scan through all of your CSS codes and refactor them to elimiate some codes which can be achieved by bootstrap, since the main purpose of this project is to learn and master bootstrap.

- [x] Using IDs as CSS selectors is a bad practice for CSS codes, please focus on using classes instead.

- [x] Using `!important` in your CSS codes, is also a bad practice. Please remove them for the sake of easy maintainability of your codes.
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested 💀 

Thank you for submitting this Pull Request. You have done a great job so far! 👏 

There is room for improvement in the code. I would recommend following these suggestions to make your project outstanding.

- [x] Each `href` should have `#` as a link. Empty `href` makes code dirty. 
- [x] For many instances you have used CSS. You can always find a replacement of CSS in bootstrap classes. For this project, your concern should be using Bootstrap so Please remove CSS and use Boostarp classes. 

When you are done with these changes request another review.  🧻 

**Best,
Arslan Bisharat**
In case of any question please reach out to me at @arslanbisharat on slack 📞

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Henry Onu [status: Enrolled] & Maksymilian Zuzel** 
Good job with the improvements for this project :muscle:  :muscle:
Most of your page looks like the original :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [x] On the Editor's pick section, each image must have a link
- [x] On the Editor's pick section, each title must have both an hx tag and a link
- [x] On tablet screen size, the Editor's pick section should organize into two columns
- [x] On the Main newspaper sections (US, World, etc.), each image should have a link
- [x] On the Main newspaper sections (US, World, etc.), each title must have both an hx tag and a link
- [x] On subscribe section, for mobile screen size, the text should overlap the image in this section:

| Yours | Original |
|:---:|:---:|
|image | image |
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Henryhaulka and @MakZ :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by Henryhaulka** [comment]
Project Approved

**Pull request merged**